kobiton-rest-api/get-devices-info/README.md

```diff
@@ -0,0 +1,147 @@
+# Getting device(s) information
+
+**Table of contents**
+- [Getting device(s) information](#getting-devices-information)
+  - [A. Overview](#a-overview)
+  - [B. Preparations](#b-preparations)
+    - [1. Prerequisites](#1-prerequisites)
+    - [2. Getting Kobiton Username and API Key](#2-getting-kobiton-username-and-api-key)
+    - [3. Getting Kobiton Group ID](#3-getting-kobiton-group-id)
+  - [C. Samples](#c-samples)
+    - [1. Get all devices](#1-get-all-devices)
+    - [2. Getting online devices](#2-getting-online-devices)
+    - [3. Getting online Android/iOS devices](#3-getting-online-android-ios-devices)
+    - [4. Getting online Galaxy S8 and running Android 7.0](#4-getting-online-galaxy-s8-and-running-android-70)
+
+## A. Overview
+This guideline will provide instructions and sample cases to demonstrate the consuming of `/devices` endpoint in Kobiton REST API.
+
+> More information about Kobiton REST API `/devices` endpoint can be found at [Api documentation](https://api.kobiton.com/docs/#devices)
+
+## B. Preparations
+### 1. Prerequisites
+- cURL installed.
+  - `Windows`: Visit https://curl.haxx.se/windows/ for instructions on how to download and install cURL on Windows.
+  - `macOS`: Installed by default
+
+### 2. Getting Kobiton Username and API Key
+Kobiton Username and API key are required for authenticating with Kobiton API.
+
+> If you don't have a Kobiton account, visit https://portal.kobiton.com/register to create one.
+
+To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script for Kobiton` section on [our blog](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/).
+
+### 3. Getting Kobiton Group ID
+If your account is in an organization and assigned to one or more groups, you will need to get the group ID.
+
+Follow instructions at [Kobiton documentation](https://docs.kobiton.com/organization-management/automation-for-groups/default-group-setting/) to get your group ID.
+
+## C. Samples
+
+In this section, we will provide sample use cases for `/devices` endpoint using bash commands with cURL.
+
+- When execute the commands provided in this section, remember to replace these variables with your values:
+  - `<YOUR_KOBITON_USERNAME>`: Your Kobiton Username.
+  - `<YOUR_KOBITON_API_KEY>`: Your Kobiton API Key.
+  - `<YOUR_GROUP_ID>`: Your group Id
+
+- Expected output for all samples in this section:
+```javascript
+{"privateDevices":[...],"favoriteDevices":[...],"cloudDevices":[...]}
+```
+
+>**Note**: If you are in org and in groups but you do not specify param `groupId` when running below commands, Kobiton REST API will fetch all devices info in all group that you are assigned in.
```

Comment:
If you are in an organization and groups but you do not specify the param `groupID` when running commands below, Kobiton REST API will fetch all devices that you can access, including devices in all groups that you were assigned and your private device (if any)